src/@next/Modal/Modal.tsx

```diff
@@ -53,6 +53,7 @@ export const Modal = React.forwardRef<HTMLDivElement, ModalProps>(
       closeOnClickOutside,
       onClose,
       onBack,
+      ...props
```

Comment:
Don't you need to extend the ModalProps component with DivElement? Or did forwardRef took care of it?

Reply:
I don't think so, i will add it

Reply:
Added